core-services/image-mirroring/openshift/mapping_origin_scos-4_18

```diff
@@ -1,202 +1,9 @@
-registry.ci.openshift.org/origin/scos-4.18:agent-installer-api-server quay.io/openshift/origin-agent-installer-api-server:4.18 quay.io/openshift/origin-agent-installer-api-server:4.18.0
-registry.ci.openshift.org/origin/scos-4.18:agent-installer-csr-approver quay.io/openshift/origin-agent-installer-csr-approver:4.18 quay.io/openshift/origin-agent-installer-csr-approver:4.18.0
```

Comment:
should we leave these as is or will they be added back when the 4.18 configs are added?

Reply:
They will be added back with follow-up 4.18 PR.